### PR TITLE
Sidebar Single widget updates & fixes

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: layouts/partials/sidebar-single/sidebar.css [hunk 1]
```diff
@@ -10,6 +10,10 @@
     {{ end }}
 "}" +
 
+"#{{ ($.Scratch.Get "settings").name }}.iframe-content{"+
```
Comment (rfornes) [inline layouts/partials/sidebar-single/sidebar.css]: Why do we need the id selector here?

Comment (rfornes) [reply layouts/partials/sidebar-single/sidebar.css]: @karloscastaneda

Comment (karloscastaneda) [reply layouts/partials/sidebar-single/sidebar.css]: @rfornes this was specifically for the jumpsport widget, *view more* button was getting chopped.

note: there was no ticket.

Comment (rfornes) [reply layouts/partials/sidebar-single/sidebar.css]: Why do we need the id selector here?

Comment (karloscastaneda) [reply layouts/partials/sidebar-single/sidebar.css]: ah yeah, the actual id selector is not necessary, will remove it.

Comment (rfornes) [reply layouts/partials/sidebar-single/sidebar.css]: :+1: please let me know once this has been removed thanks.

Comment (karloscastaneda) [reply layouts/partials/sidebar-single/sidebar.css]: @rfornes updated

File: static/sidebar-single/js/grid.js [hunk 2]
```diff
@@ -148,7 +150,7 @@
       if (that.windowSize !== newSize) {
         that.windowSize = newSize;
         var isSmall = newSize === 'small';
-        that.itemsPerPage = isSmall ? 2 : (options.itemsPerPage || 6);
+        that.itemsPerPage = isSmall ? 2 : (options.items_per_page || 6);
```
Comment (rfornes) [inline static/sidebar-single/js/grid.js]: This should read:

**that.itemsPerPage = isSmall ? 2 : options.items_per_page;**

`items_per_page` should exists in the `defaults/` folder and it should have a default value set (6 i guess)

**Pull request closed without merging**